internal/service/resource_handler.go

```diff
@@ -412,12 +412,12 @@ func (r *ResourceHandler) mapNodeItem(ctx context.Context, // nolint: unparam
 	}
 
 	to = data.Object{
-		"resourceID":     resourceID,
-		"resourceTypeID": resourceTypeID,
+		"resourceId":     resourceID,
+		"resourceTypeId": resourceTypeId,
 		"description":    description,
 		"extensions":     extensionsMap,
-		"resourcePoolID": resourcePoolID,
-		"globalAssetID":  globalAssetID,
+		"resourcePoolId": resourcePoolId,
+		"globalAssetId":  globalAssetID,
```

Comment:
Should probably change the globalAssetID variable to match as well to be consistent with the others.